dotnet/src/webdriver/BiDi/Modules/Script/LocalValue.cs

```diff
@@ -41,10 +41,10 @@ namespace OpenQA.Selenium.BiDi.Modules.Script;
 [JsonDerivedType(typeof(SetLocalValue), "set")]
 public abstract record LocalValue
 {
-    public static implicit operator LocalValue(bool? value) { return value is bool b ? new BooleanLocalValue(b) : new NullLocalValue(); }
-    public static implicit operator LocalValue(int? value) { return value is int i ? new NumberLocalValue(i) : new NullLocalValue(); }
-    public static implicit operator LocalValue(double? value) { return value is double d ? new NumberLocalValue(d) : new NullLocalValue(); }
-    public static implicit operator LocalValue(string? value) { return value is null ? new NullLocalValue() : new StringLocalValue(value); }
+    public static implicit operator LocalValue(bool? value) { return ConvertFrom(value); }
+    public static implicit operator LocalValue(int? value) { return ConvertFrom(value); }
+    public static implicit operator LocalValue(double? value) { return ConvertFrom(value); }
+    public static implicit operator LocalValue(string? value) { return ConvertFrom(value); }
 
     // TODO: Extend converting from types
     public static LocalValue ConvertFrom(object? value)
```

Comment:
We already have unit tests for this method. I propose to create some universal test to demonstrate the power of conversion. Where user is able to properly convert any "free-style object" to `LocalValue` (including all supported complex types). The idea behind it is:
- double-check for us
- see all supported types in one place
- easily understand what we are missing

Reply:
I fixed the conversion, including for `Set`. Now it is time just prepare tests.

Reply:
Added minimal tests, let's go.